system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -6232,7 +6232,7 @@ class CLdapClient : implements ILdapClient, public CInterface
         StringBuffer resName(queryDfsXmlBranchName(DXB_Internal));
         resName.append("::").append(user.getName());
         Owned<ISecResource> resource = new CLdapSecResource(resName.str());
-        return addResource(RT_FILE_SCOPE, user, resource, PT_ADMINISTRATORS_AND_USER, m_ldapconfig->getResourceBasedn(RT_FILE_SCOPE));
+        return addResource(RT_FILE_SCOPE, user, resource, PT_ADMINISTRATORS_ONLY, m_ldapconfig->getResourceBasedn(RT_FILE_SCOPE));
```

Comment:
Similarly, don't think this place applies, because this is within an operation specifically for a certain user.